### PR TITLE
Ion Storm Change

### PR DESCRIPTION
**Pull request by Dragonouhai3**
Changes ion storm to be more australian

## Changelog
:cl:
add: new ionstorm sound file, renamed old one to ionstorm (Original)
code: change ionstorm event code
:cl:

### CODE REVIEW

File: code/modules/events/ion_storm.dm [hunk 1]
```diff
@@ -10,7 +10,7 @@
 	var/removeDontImproveChance = 10 //chance the randomly created law replaces a random law instead of simply being added
 	var/shuffleLawsChance = 10 //chance the AI's laws are shuffled afterwards
 	var/botEmagChance = 1
-	var/ionMessage = null
+	var/ionMessage = "Telstra is having an unexpected maintenance on their systems, we apologies for any issues caused."
```
Comment (MCterra10) [inline code/modules/events/ion_storm.dm]: Grammar and spelling, also modularize this

File: code/modules/events/ion_storm.dm [hunk 2]
```diff
@@ -23,7 +23,8 @@
 
 /datum/round_event/ion_storm/announce(fake)
 	if(prob(announceChance) || fake)
-		priority_announce("Ion storm detected near the station. Please check all AI-controlled equipment for errors.", "Anomaly Alert", 'sound/ai/ionstorm.ogg')
+		//priority_announce("Ion storm detected near the station. Please check all AI-controlled equipment for errors.", "Anomaly Alert", 'sound/ai/ionstorm.ogg')
+		priority_announce("Telstra is having an unexpected maintenance on their systems, we apologies for any issues caused.", "Anomaly Alert", 'sound/ai/ionstorm.ogg')
```
Comment (MCterra10) [inline code/modules/events/ion_storm.dm]: Grammar and spelling, also modularize this

Comment (Dragonouhai3) [reply code/modules/events/ion_storm.dm]: just noticed I screwed up that spelling, oops...

**Comment by MCterra10** [comment]
author informed me they are no longer working on this PR. If you start work again, make a new PR with changes

**Pull request closed without merging**